source/designers-guide/how-to-use-karma/index.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: How to use Karma for storefront tests
+github_link: designers-guide/how-to-use-karma/index.md
+shopware_version: 5.0.3
+indexed: true
+---
+
+## Introducing
+[Karma](http://karma-runner.github.io/) is a test runner for JavaScript tests for the storefront of Shopware. The tests itself are written in [Jasmine](http://jasmine.github.io/).
```

Comment:
[Karma](http://karma-runner.github.io/) is a JavaScript test runner for Shopware's storefront. The tests themselves are written in [Jasmine](http://jasmine.github.io/).